traffic_portal/app/src/common/modules/form/_form.scss

```diff
@@ -141,3 +145,20 @@ dl dd {
 	margin-left: 50px;
 }
 
+/********** styling for showing diffs on a delivery service request **********/
+
+aside.current-value {
+	border: 1px solid #ddd !important;
+	margin: 10px;
+	padding: 10px;
+	position: relative;
+	border-radius: 4px;
+	background-color: #f5f5f5;
+	padding-left: 10px !important;
```

Comment:
were `border` and `padding-left` being overridden by something?

Reply:
nope. i'll fix that. i was just copying some stying from the fieldset